src/uct/cuda/cuda_copy/cuda_copy_md.c

```diff
@@ -286,52 +439,119 @@ static size_t uct_cuda_copy_md_get_total_device_mem(CUdevice cuda_device)
     return 1; /* return 1 byte to avoid division by zero */
 }
 
+static ucs_status_t
+uct_cuda_copy_detect_vmm(const void *address, CUmemorytype *cuda_mem_type,
+                         unsigned *is_vmm_allocation)
+{
+    ucs_status_t status = UCS_OK;
+#if HAVE_CUDA_FABRIC
+    CUmemAllocationProp prop = {};
```

Comment:
do we have to initialize?

Reply:
Same as the comment here https://github.com/openucx/ucx/pull/9867#discussion_r1598881045